CHANGES.rst

```diff
@@ -121,6 +121,8 @@ Other Changes and Additions
 Bug Fixes
 ---------
 
+- Fix aperture photometry issue for cubes in deconfigged. [#3897]
+
```

Comment:
not sure this needs a changelog since it was never released (or can add to the existing cube support entry if there is one)